src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -1,73 +1,109 @@
+                  <div className="usa-alert-body">
+                    <p><strong>Disability ratings</strong></p>
+                    <p>We assign disability ratings with a severity from 0% to 100% for each disability claim. This rating can change if there are changes in your condition. We’ll decide a claim for increase based on the medical evidence and supporting documents you turn in with your claim.</p>
+                    <p><a href="/disability-benefits/eligibility/ratings/">Learn how VA assigns disability ratings</a>.</p>
+                  </div>
+                </div>
+                <br/>
+              </div>
             </li>
             <li className="process-step list-two">
               <div><h5>Apply</h5></div>
-              <p>Complete this education benefits form.</p>
+              <p>Complete this disability compensation benefits form.</p>
               <p>After submitting the form, you’ll get a confirmation message. You can print this for your records.</p>
             </li>
             <li className="process-step list-three">
               <div><h5>VA Review</h5></div>
-              <p>We usually process claims within 30 days. We’ll let you know by mail if we need more information.</p>
-              <p>We offer tools and counseling programs to help you make the most of your educational options. <a href="/education/tools-programs/">Learn about career counseling options.</a></p>
+              <p>We usually process claims within <strong>103 days</strong>.</p>
             </li>
             <li className="process-step list-four">
               <div><h5>Decision</h5></div>
-              <p>You’ll get a Certificate of Eligibility (COE), or award letter, in the mail if we've approved your application. Bring this to the VA certifying official at your school.</p>
-              <p>If your application wasn’t approved, you’ll get a denial letter in the mail.</p>
+              <p>Once we’ve processed your claim, you’ll get a notice in the mail with our decision.</p>
             </li>
           </ol>
         </div>
-        <SaveInProgressIntro
+        <FormStartControls
+          user={user}
+          authenticate={this.authenticate}
+          ITFStatus={ITFStatus}
+          {...this.props}
           buttonOnly
-          messages={this.props.route.formConfig.savedFormMessages}
-          pageList={this.props.route.pageList}
-          startText="Start the Education Application"
-          {...this.props.saveInProgressActions}
-          {...this.props.saveInProgress}/>
+          beforeStartForm={this.beforeStartForm}/>
         {/* TODO: Remove inline style after I figure out why .omb-info--container has a left padding */}
         <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
-          <OMBInfo resBurden={20} ombNumber="2900-0074" expDate="05/31/2018"/>
+          <OMBInfo resBurden={25} ombNumber="2900-0747" expDate="11/30/2017"/>
```

Comment:
Is this expiration date correct?

...Déjà vu. Did I already ask that?

Reply:
This information matches that provided on the PDF version of the form, that we link on the site, looping in @peggygannon and @goldenmeanie in case, that PDF shouldn't be the basis of this information (which is also on the prototype).

Reply:
Heh, turns out I did already ask this! Sorry for repeating myself! :grimacing: